app.js

```diff
@@ -90,41 +90,43 @@ sitemap(config, function (sitemap, urls) {
 	
 	var results = _.map(urls, function (url, index) {
 		console.log('Registered ' + url.url);
-		var processResults = processOne(config, url, function (error, record) {
-			if (!!error || !record) {
-				console.error('Error! ' + error.message);
-				if (!!error.pageNotFound && !!record) {
-					pages.deleteObject(record.objectID, function (error, result) {
-						console.log('Object ' + record.objectID + ' has been deleted');
-					});
-				}
-				removeOldEntries();
-				return;
-			}
-			
-			pages.saveObject(record, function (error, result) {
-				if (!!error) {
-					console.log();
-					if (!!result && !!result.message) {
-						console.error('Error! ' + result.message);
-					}
-					if (!!error && !!error.message) {
-						console.error('Error! ' + error.message);
+		setTimeout(function () {
```

Comment:
I think that the timeout should be in the `processOne()`, WDYT ?

Reply:
In my opinion, I prefer the code on the setTimeout out of the processOne beacause this setTimeout is intended for a whole not a single one. wdyt?

Reply:
No it is intended to be a delay between each request to a page. We just need to delay everything in the processOne function. It will first wait, then do things.